unistd/netdb.c

```diff
@@ -0,0 +1,15 @@
+// netdb.c
+// Provided by brimdata on 2022/10/19
+// Copyright (c) 2019 Robin.Rowe@CinePaint.org
+// License open source MIT
+
+#include "netdb.h"
+#include <stdlib.h>
+
+struct hostent *gethostbyname2(const char *name, int af)
+	{
```

Comment:
I'm going to see about getting a clang-format configuration committed upstream so that we can reformat things to match some sort of standard. For example, this file is in Whitesmiths, but lots of others are in other formats.